src/main/java/seedu/clinic/logic/commands/UpdateCommand.java

```diff
@@ -74,7 +74,7 @@ public CommandResult execute(Model model) throws CommandException {
         model.setWarehouse(warehouseToUpdate, updatedWarehouse);
         model.updateFilteredWarehouseList(PREDICATE_SHOW_ALL_WAREHOUSES);
         return new CommandResult(String.format(MESSAGE_SUCCESS, updatedProduct.toStringForWareHouse(),
-                updatedWarehouse));
+                updatedWarehouse.getName().fullName));
```

Comment:
Not too sure if it makes a major difference, but I am wondering if it would be better to access the fullName via some form of getter method instead of accessing the attributes directly. 🤔

Reply:
Hmm not sure either...full name is originally a public attribute with no getters, and this `.fullname` is used is several places in the original code as well. Same for the other supplier/warehouse properties.

Reply:
Hmm, I guess in that case let's just use `.fullname` instead of a getter method then, since it is used in several places as well :)